empirical_photometry.ipynb

```diff
@@ -0,0 +1,448 @@
1Lhw4dqF69On///TebN29m48aN0pOjI5PX2VFKXQGGapr2NXEzsboBP8dPS18K/KCUCjdPTCGEEHrYuHEjmTJlom7duowcOVK3H9j79++ne/fu3Lp1Cx8fHyZOnEiGDKZttGxpN27cYPPmzYwaNYphw4ZZbc7U5L/0P9oBTsRNQ7cBbgEdgVuaprVPwmxCCCF08uTJE7y9vWnTpg3z5s0D9Nml/NGjR/Tq1YvatWuTNm1aDh8+zMyZM62ugNi3bx9Tp04FoFKlSoSEhDBu3Diry5lamVzsaJrmqWna98A9YBpwAiiqlKqllHIlbi2emeaJKYQQwlKCgoLw9PRkyZIlDBs2jE2bNumSY8+ePbi5ubFkyRKGDBnCuXPnqFy5si5Z3ubWrVu0atWKOnXqsHTpUqKiogDInj27zsnEy0ydjXUeOAbkA7oAzkqpEUqp6y81+xGwzu1jhRBCmCQ4OJjy5cvz119/8fPPPzN58mTs7OwsmiEiIoJu3bpRr149HBwcOHbsGNOmTSN9+vQWzZGYp0+fMnHiREqUKMHOnTsZP348QUFBVpVR/MvUnp0NQEGlVGOl1DallPH1Bkqph0opWfoxGTl48CCaprFixQq9owghdBYbGwuAi4sLY8eOJSgoiNq1a1s8h7+/P66urqxatYqvv/6as2fPUqFCBYvneJd79+4xYcIEGjRowKVLlxg5ciTp0qXTO5Z4C1NnY41XSt0xdxghhBCWd/DgQdzc3Pjjjz/QNI2hQ4eSI4dlO+rDw8Pp1KkTjRs3JmvWrJw4cYJJkyZZVQHxxx9/MGHCBJRSFCxYkEuXLuHr64uzs7Pe0cQ7mDQbS9O0ZW85pYCnxG0Sul4pdTepggnzq1atGlFRURbvohZCWAeDwcDYsWOZOHEiRYsWfTHexNK2bt1Knz59CA0NZfTo0YwYMQJ7e3tdsiTkyZMnTJkyhWnTpmFra0v79u0pVKgQBQoU0DuaMJGpU89zAFWBWOBC/DE3QAPOAC2AcZqmVVVKnUvylMIs0qRJY1X/ahJCWM6NGzfo0KEDx44do2vXrsyZM8fi69WEhobSv39/fvrppxerMnt4eFg0Q2KUUmzdupUvv/ySmzdv0r59e6ZNm0aePHn0jibek6ljbI4Cu4C8SqlqSqlqQF5gJ/AzcZuA7gBmmCVlChB96xb3xo7lSllPLpV04UpZT+6NHUv0rVu6ZZIxO0KkXjNnzuT8+fP8+OOPLFu2zOKFjq+vL66urvj6+jJu3DhOnTplVYUOxG1J0adPHxwdHTl48CBr166VQieZMrXYGQCMU0o9eX4g/vlE4EulVDQwFbCub6qVeHzoENeaNiNioy+xkZGgFLGRkURs9OVa02Y8PnRI74hCiFTgyZMnXLt2DYDJkycTGBhIu3btLJrhwYMHtG7dmtatW5MvXz7OnDnDqFGjrOZ2+uPHj5kxYwYxMTE4ODhw4MABAgICqF5dNgxIzkwtdhyADxM4niv+HMAj3mNF5tQi+tYtbg/wQUVFgcHw6kmDARUVxe0BPrr28AghUr7AwEA8PT1p1KgRBoOBDBkyULBgQYtdXynFunXrcHFxYdu2bUyaNIkTJ05Yza7fSinWr19PiRIlGDx4ML/88gsAJUuWxNZWfrQld6YWO1uApZqmtdY0rYCmac6aprUmbpuIzfFtygNXzREyOQtbvhwVE5NoGxUTQ9iKlRZKJIRITZRSzJs3jwoVKhAREcHcuXMt/sP7/v37tGjRgnbt2lG4cGECAgL4+uuvraaIuHDhAjVr1uTTTz/lgw8+4NixY3zyySd6xxJJyNRipzewB1gD/AFci3++G/g8vs0loGdSB0zuHm3b/maPzusMBh5t22aZQEKIVOPRo0c0bdqUL774glq1ahEYGEitWrUsdn2lFKtXr8bFxYVdu3Yxbdo0jh49iouLi8UyvItSik6dOhEUFMTChQs5efIkFStW1DuWSGImldXx43N6a5o2CChM3Cys/ymlIl9qI7OwEhD75Mm7G0HcWB4hhEhC6dOn59GjR8yaNYv+/ftbdG+rO3fu0Lt3b/z9/alUqRLLli2jePHiFrt+YmJjY/nxxx9p1KgRmTNnZs2aNXzwwQdky5ZN72jCTN5rxWOlVKRSKkgpFfhyoSPeLo2Jm8ClyZjRzEmEEKmBwWBgypQphIaGYmdnxy+//MKAAQMsVugopVi+fDmurq7s37+fmTNncujQIaspdM6dO0fVqlXp2LEjS5cuBeJWjZZCJ2UzdVHBA8QtIPi6lxcVXKmUOpuE2VIEpyaNidjom/itLFtbnJo0sVwoIUSK9PLaOU5OTnz++eekSWO5XXxCQkLw9vZm9+7dVKtWjaVLl1KkSBGLXT8x4eHhjBo1ioULF5ItWzaWLVtG586d9Y4lLMTUvwWXgDLEzci6Hf/4MP7YA6AK8JumaZa7GZxMZOvaFe0dUyo1OzuydZG/dEKI/27jxo14eHhw4cIFfvzxRz7//PN3vyiJKKVYvHgxrq6uHD58mLlz53LgwAGrKXQAvvjiCxYuXEjfvn25cuUKXbt2tWghKPRl6p/0U2CFUqqkUqpT/KMksAwIU0qVBb4HJpgraHJlnz8/eWfPQkufHl6feWBri5Y+PXlnz8I+f359Agohkr358+fTpk0bSpQoQUBAgEXXzrlx4wZ169bF29sbT09Pzp8/T79+/ayikDh58iQ3btwAYPz48Zw9e5Y5c+aQJUsWfYMJizN13l9n4OMEji8CjgODgR+ALkkTK2VxqFaNQn5bCVuxkkfbthEbGUmajBlxatKEbF0661boeHl5oVRCdyeFEMmBUgpN02jdujXh4eEMGzbMYovzxcbGsmjRIr766isAFixYgLe3t1UUOQ8fPmT48OEsXbqUzp07s3z5cgoVKqR3LKEnpdQ7H0A40CyB482A8PjnxYC/EnmPvkAQcYsPPiKuSGr40nkNGAPcBaKAg4Dra++RBVgN/B3/WA1kNuUzKKUoW7asep
```

Comment:
Minor: could you make the plot square so that the 1:1 line is at a 45 deg angle to please a tiny part of my brain?